client/ayon_hiero/plugins/publish/collect_plates.py

```diff
@@ -16,6 +16,30 @@ def process(self, instance):
         """
         instance.data["families"].append("clip")
 
+        # solve reviewable options
+        review_switch = instance.data["creator_attributes"].get("review")
+        review_track_name = instance.data["creator_attributes"].get(
+            "reviewableTrack")
+
+        if review_switch is True:
+            instance.data["families"].append("review")
+            instance.data.pop("reviewTrack")
+
+        if (
+            review_track_name != "< none >"
```

Comment:
I think review track name here cannot be `"< none >"`.

Reply:
8cbf30c2b1e81d73c2b5a8bf473e1bd5e39800a5, yes you are right, it is fixed